ts/Data/DataPoolOptions.ts

```diff
@@ -33,8 +33,22 @@ export interface DataPoolOptions {
 
 export interface DataPoolConnectorOptions
 <T extends keyof DataConnectorTypes = keyof DataConnectorTypes> {
+    /**
+     * The unique identifier of the connector. Used later when referencing
+     * the connector in the component where it is used.
+     **/
     id: string;
+    /**
+     * The options of the given connector type.
+     **/
     options: DataConnectorTypes[T]['prototype']['options'];
+    /**
+     * The type of the connector, depends on your data source.
+     * Possible values are:
+     * - `CSV`
+     * - `GoogleSheets`
+     * - `HTMLTable`
```

Comment:
Isn't the bundler show all available types?

Reply:
No, there is no registry for that.

Reply:
`DataConnector.types` is it.

Reply:
+1

Reply:
DataConnector registry is not ready for API docs. So we leave it for now.